src/OSPSuite.Core/Domain/Constants.cs

```diff
@@ -14,6 +14,8 @@ public static class Constants
       public const int MAX_NUMBER_OF_POINTS_PER_INTERVAL = 5000;
       public const int MIN_NUMBER_OF_POINTS_PER_INTERVAL = 2;
       public const int MAX_NUMBER_OF_CHAR_IN_TABLE_NAME = 29;
+      public const string WILD_CARD = "*";
+      public const string WILD_CARD_REC = "**";
```

Comment:
REC="Recursive"?

Reply:
@Yuri05 Yep. Pretty standard shortcut but I can expand if you prefer

Reply:
done